models/comments.model.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../db/connection.js')
+const format = require('pg-format')
+
+exports.uploadComment = (comment, review_id) => {
+    if(!review_id) {
+       return Promise.reject({status: 400, msg: "Invalid review ID"})
+    }
+    if(!comment.body){
+        return Promise.reject({status: 400, msg:'Comment is missing a body'})
+    } 
+    
+    return db.query(`SELECT username FROM users WHERE username = $1`, [comment.username])
+    .then( result => result.rows)
+    .then( users => {
```

Comment:
author in the comments table is a foreign key which relates to username in the users table. A psql error will be thrown if this is violated. It is not necessary to separately check if the user exists.
article_id is also a foreign key so again, a separate check that the article exists should not be necessary as a psql error will be thrown.
a psql error will also be thrown if article_id is not a number.